README.md

```diff
@@ -83,6 +83,22 @@ We're using PouchDB to create an in-browser database via the Indexed DB API. Usa
 ### CSS
 The header/main nav and PLP page are using CSS Grid and Flexbox. CSS Grid is amazing to work with, so exploration of these is encouraged! We don't need to support IE!
 
+### Automated testing with Pa11y
+An automated accessibility test can be run to look for major accessibility or HTML validity errors. The test is run using pa11y-ci (https://github.com/pa11y/pa11y-ci).
+
+Before testing, please run (or stop and re-run)
+```bash
+npm start
+```
+and then Control+C to stop the Parcel server. This will ensure the files are all up to date.
```

Comment:
We should be able to just do `npm run build`, then the files in `/dist` will be generated and the server won't be started (or need to be stopped)